dotcom-rendering/src/web/browser/ga/ga.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/strict-boolean-expressions, @typescript-eslint/no-unnecessary-condition --
+	We check the existence of window.ga frequently, which is required for browser safety despite the type definitions. */
```

Comment:
Should we make it optional in our type definitions?

Reply:
It's a bit confusing to me... window.guardian.ga is defined in our types and is optional, I think this is probably worth looking at in a new PR